src/status_im/contexts/wallet/send/utils.cljs

```diff
@@ -24,6 +25,23 @@
              {}
              transaction-hashes))
 
+(defn calculate-gas-fee
```

Comment:
When I see this function in utils, I have an impression that it can be reused from other namespaces but it is not clear what `data` should be passed.
If it shouldn`t be used in other namespaces I suggest to make it private and name more close to the meaning, something like 
`(defn- calculate-gas-fee-for-route [route])`

Reply:
I agree we can improve naming, I think I would prefer to keep the function public so we can test it independently. However, I'd prefer to rename it in a separate PR, as here I am just moving this function to other utils file.